packages/eslint-config-react/package.json

```diff
@@ -28,7 +28,8 @@
     "@socialgouv/eslint-config-recommended": "^0.5.2",
     "babel-eslint": "^10.0.1",
     "eslint-plugin-jsx-a11y": "~6.2.1",
-    "eslint-plugin-react": "~7.12.4"
+    "eslint-plugin-react": "~7.12.4",
+    "eslint-plugin-react-hooks": "^1.6.0"
```

Comment:
As linting rules can break builds, IMO it's better to use `~` than `^`

Reply:
updated !